### DISCUSSION TITLE
Code coverage

### DISCUSSION THREAD

**Discussion by steve-lgtm** [mathesar-foundation/mathesar#2018] [Developer Q&A] [answered]
Hi, how can I run code coverage locally?

**Comment by kgodey** [comment, marked as answer]
[Here's the commend to run coverage](https://github.com/centerofci/mathesar/blob/master/.github/workflows/run-pytest.yml#L39) from our CI workflow, you can also use it locally. 

We use `pytest-cov`, you can also read through [their documentation](https://pytest-cov.readthedocs.io/en/latest/reporting.html) to generate reports in other formats, XML is probably not the most useful locally.

**Reply by steve-lgtm** [reply]
but that is for backend only ? I need coverage for fronted.

**Reply by kgodey** [reply]
We do not have any code coverage reporting set up for the frontend.
